ENIGMAsystem/SHELL/Graphics_Systems/General/GSvertex.h

```diff
@@ -20,6 +20,7 @@
 #define ENIGMA_GSVERTEX_H
 
```

Comment:
Assuming my above statements are correct, could you do me a favor and document that flow, here? Something like this:

```c++
// This file declares methods for translating generic vertex and index
// buffer representations to the backend equivalents (e.g, OpenGL).
// Systems which would like to offer support for ENIGMA's vertex
// functions need only to implement these methods.
```

Reply:
Yes, but perhaps add clarification that right now only the following 4 must be implemented for the general ones to work:
* `vertex_argb`
* `vertex_color`
* `vertex_submit`
* `index_submit`

Reply:
Even better. Please do.

Reply:
Done: 625ebced5bb08f6c7e84bedfc72a95236b569990